server/libs/database/src/entities/asset.entity.ts

```diff
@@ -44,6 +44,9 @@ export class AssetEntity {
   @Column({ type: 'varchar', nullable: true })
   mimeType!: string | null;
 
+  @Column({ type: 'char', length: 40, nullable: true })
```

Comment:
Yeah, hex string is feasible to search/index. As pin from discord, mobile may ask server for duplication with hash they calculated. So hex string would be useful in this case.  I don't think we can search `bytea` easily, do I?

Reply:
In case the mobile app sends a hash to be checked, it is simply converted from hex string to bytes for searching in the DB or cache. I am confident postgres can index bytes as well

Reply:
@zoodyy ok, I will try with `bytea`